feature/experimental/qos/ate_tests/qos_output_queue_counters_test/qos_output_queue_counters_test.go

```diff
@@ -547,6 +547,11 @@ func ConfigureQoS(t *testing.T, dut *ondatra.DUTDevice) {
 		s := schedulerPolicy.GetOrCreateScheduler(tc.sequence)
 		s.SetSequence(tc.sequence)
 		s.SetPriority(tc.priority)
+		if dut.Vendor() == ondatra.JUNIPER {
+			t.Logf("Scheduler should be binded to a forwarding group")
+			Output := s.GetOrCreateOutput()
+			Output.SetOutputFwdGroup(tc.targetGroup)
+		}
```

Comment:
Please add the tracking bug for this change.

Reply:
As agreed, make it gated by deviation